runtime/Cpp/runtime/src/atn/LexerActionType.h

```diff
@@ -14,7 +16,7 @@ namespace atn {
   /// @author Sam Harwell
   /// @since 4.2
   /// </summary>
-  enum class ANTLR4CPP_PUBLIC LexerActionType : size_t {
+  enum class LexerActionType : size_t {
```

Comment:
No, "enum class" defines compile time constants, not external symbols. Those symbols are not exported from a DLL like a real class, so specifying linker behaviour for them is meaningless.